### PR TITLE
[manual] Merge release/6.0-staging into release/6.0

### PR DESCRIPTION
**Pull request by carlossanlop**
**Merge commit.**

### CODE REVIEW

File: docs/design/features/dotnet-pgo.md [hunk 1]
```diff
@@ -37,14 +29,14 @@ set COMPLUS_TC_QuickJitForLoops=1
 set COMPLUS_TC_CallCountThreshold=10000
 set COMPLUS_ReadyToRun=0
 
-dotnet-trace collect --providers Microsoft-Windows-DotNETRuntime:0x1E000080018:4 -- bin\Release\net6.0\pgotest.exe
+dotnet-trace collect --providers Microsoft-Windows-DotNETRuntime:0x1E000080018:4 -- bin\Release\net{version-number-goes-here}.0\pgotest.exe
```
Comment (vseanreesermsft) [inline docs/design/features/dotnet-pgo.md:32]: does this {version-number-goes-here} get substituted later on?

**Review by vseanreesermsft — APPROVED** [review]
lgtm

**Comment by carlossanlop** [comment]
FYI OOB packages from last month have been reset directly here instead with a separate PR: https://github.com/dotnet/runtime/pull/104925/commits/291338675fca08a7ba73f36be1a13f5e2f9d7d19

Once merged into the base branch, An automatic PR will flow this into the staging branch.

File: docs/design/features/dotnet-pgo.md (hunk 1 above)
Comment (carlossanlop) [reply docs/design/features/dotnet-pgo.md:32]: I think it's inconsequential as it is just a markdown file. Introduced here: https://github.com/dotnet/runtime/pull/103927/files

**Pull request merged**